egs/wsj/s5/steps/compute_vad_decision.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash 
+
+# Copyright    2017  Vimal Manohar
+# Apache 2.0
+
```

Comment:
It might be useful for segmenting long utterances when there is a lot of silence in the recordings. I have not tried using without this VAD. But it needs to be tested.